wrapanapi/systems/msazure.py

```diff
@@ -761,6 +768,24 @@ def remove_netsec_group(self, group_name, resource_group=None):
         self.logger.info("Network Security Group '%s' is removed", group_name)
         return operation.status()
 
+    def create_netsec_group_port_allow(self, group_name, port, resource_group=None):
+        secgroup_name = 'port_{:d}'.format(port)
+
+        self.logger.info("Attempting to Create New Azure Security Group '%s' "
+                         "Rule '%s'.", secgroup_name, group_name)
+
+        parameters = NetworkSecurityGroup(location=self.region)
+        parameters.security_rules = [
+            SecurityRule(
+                'Tcp', '*', '*', 'Allow', 'Inbound', description='Allow port {:d}'.format(port),
```

Comment:
I think this method will be a lot more useful if the rule attributes were taken as a parameter instead of being hardcoded.